api/lib/infrastructure/repositories/organization-participant-repository.js

```diff
@@ -6,6 +6,20 @@ const CampaignTypes = require('../../domain/models/CampaignTypes');
 const CampaignParticipationStatuses = require('../../domain/models/CampaignParticipationStatuses');
 
 async function getParticipantsByOrganizationId({ organizationId, page, filters = {} }) {
+  const { count } = await knex
+    .select(knex.raw('COUNT(DISTINCT "organization-learners"."id")'))
+    .from('organization-learners')
+    .join('campaign-participations', function () {
+      this.on('campaign-participations.organizationLearnerId', 'organization-learners.id').andOn(
+        'campaign-participations.deletedAt',
+        knex.raw('IS'),
+        knex.raw('NULL')
+      );
+    })
+    .where({ organizationId: organizationId, isDisabled: false })
```

Comment:
suggestion: on peut faire plus simplement:
```suggestion
    .join('campaign-participations', 'campaign-participations.organizationLearnerId', 'organization-learners.id')
    .where({ organizationId: organizationId, isDisabled: false })
    .where({ 'campaign-participations.deletedAt': null })
```

Reply:
Merci !

Reply:
[Question] On peut laisser la modification comme ça. 

Mais il me semblait qu'en SQL il était préférable de faire le plus de condition dans la jointure afin de limiter le contenu renvoyé pour ensuite filtrer sur la table principale.

where => que sur la table principal , le reste ça va dans les jointures tant que c'est possible ( il y a certainement des exceptions ) ?

Je ne suis pas assez calé sur le SQL pour être sûr de ça. 😅